docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -716,6 +789,9 @@ intentionally use resources without patches and variables to focus on the simple
                 apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
                 kind: VSphereMachineTemplate
                 name: linux-vsphere-template
+          # This will create a health check for each deployment created with the "linux-worker" MachineDeploymentClass
+          # The machineHealthCheck values will all be defaulted as no values are specified in the empty struct below.
```

Comment:
I think we should drop the `all` here and in l.808 as it currently wouldn't be the case

Reply:
This proposal is aimed at future behaviour rather than current. The idea here is that the MHC would be defaulted to be in working order which (as far as I understand) is not the current behaviour.

Reply:
I've removed the 'all' in both places, but I'm still unclear on what difference there is in meaning.

Reply:
Slight difference is that some fields are defaulted but not all (UnhealthyConditions). Of course pending discussion the issue about defaulting.